nasa-gibs/worldview
# WV-2528: Removing embed tab and updating related tests

## Description

This removes the embed tab from the share modal. Also updates all of the tests related to embed.

## How To Test

1. `git checkout wv-2528`
2. `npm run watch`
3. Click the share button.
4. Verify that the embed tab is not there.
